Content.Client/Chemistry/EntitySystems/ChemistryGuideDataSystem.cs

```diff
@@ -113,6 +113,11 @@ private void OnPrototypesReloaded(PrototypesReloadedEventArgs? ev)
                     juiceSolution);
                 foreach (var (id, _) in juiceSolution.Contents)
                 {
+                    if (!_reagentSources.ContainsKey(id.Prototype))
+                    {
+                        Logger.Error($"{entProto.Name} has invalid reagent {id.Prototype}");
```

Comment:
`'Logger.Error(string)' is obsolete: 'Use ISawmill.Error'`

Don't use obsoleted things in new code if you can avoid it